monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -3486,6 +3493,13 @@ object Task extends TaskInstancesLevel1 {
   def wander[A, B, M[X] <: Iterable[X]](in: M[A])(f: A => Task[B])(implicit bf: BuildFrom[M[A], B, M[B]]): Task[M[B]] =
     Task.eval(in.map(f)).flatMap(col => TaskGather[B, M](col, () => newBuilder(bf, in)))
 
+  /**
+    * Similar to `wander`, but bounded by specified parallelism.
+    */
+  def wanderN[A, B, M[X] <: Iterable[X]](parallelism: Int)(in: M[A])(f: A => Task[B])(
+    implicit bf: BuildFrom[M[A], B, M[B]]): Task[M[B]] =
+    Task.eval(in.map(f)).flatMap(col => TaskGatherN[B, M](parallelism, col, () => newBuilder(bf, in)))
```

Comment:
If you want to catch errors thrown by `f` call, using `Task.suspend` could be clearer and would allocate a bit fewer things. I.e.:
```
Task.suspend {
  TaskGatherN[B, M](parallelism, in.map(f), () => newBuilder(bf, in))
}
```
just leave a comment if catching from `f` is what you intend to do.